save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/index/CardAboutUs.kt

```diff
@@ -57,17 +59,30 @@ val cardAboutUs: FC<IndexViewProps> = FC { props ->
                         style = "secondary rounded-pill",
                         isOutline = false
                     ) {
-                        Navigate {
-                            to = "/${FrontendRoutes.ABOUT_US}"
-                        }
+                        navigate(to = "/${FrontendRoutes.ABOUT_US}")
                     }
                 }
             }
 
             div {
                 className = ClassName("col-6")
                 p {
-                    +"We kindly ask you not to break this service and report any problems that you will find to our Github. Please also read our"
+                    +"We kindly ask you not to break this service and report any problems that you will find to our Github."
+                }
+                Link {
+                    className = ClassName("btn btn-secondary rounded-pill")
+                    to = "https://github.com/saveourtool/save-cloud"
+                    fontAwesomeIcon(icon = faGithub)
+                    +"  Save-cloud"
+                }
```

Comment:
This is not how you should use `Link`. Use `a` for this purpose. If I am not mistaken, `Link` is used for navigation inside your `Router`

Reply:
fixed